source/Octopus.Client/Repositories/Async/TaskRepository.cs

```diff
@@ -223,5 +223,13 @@ public Task WaitForCompletion(TaskResource[] tasks, int pollIntervalSeconds = 4,
         /// <param name="pageSize">Number of items per page, setting to less than the total items still retreives all items, but uses multiple requests reducing memory load on the server</param>
         /// <returns></returns>
         public Task<List<TaskResource>> GetAllActive(int pageSize = int.MaxValue) => FindAll(pathParameters: new { active = true, take = pageSize });
+
+        public ITaskRepository LimitTo(bool includeGlobal, params string[] spaceIds)
```

Comment:
Given the name `LimitTo`, I would expect this to work:
`taskRepository.LimitTo(true, "Spaces-1", "Spaces-2").LimitTo(true, "Spaces-1")`

but not this:
`taskRepository.LimitTo(true, "Spaces-1", "Spaces-2").LimitTo(true, "Spaces-3")`

In the latter case, the space selection is not a subset of the previous repository scope, so I would expect it to throw an exception.